src/main/java/views/AcesUp/SPacesUp.flt.html

```diff
@@ -0,0 +1,314 @@
+<!DOCTYPE html>
+<html>
+<head>
+<link href="/assets/css/bootstrap.min.css" rel="stylesheet">
+<link href="/assets/css/AcesUp.css" rel="stylesheet">
+<link href="/assets/css/custom.css" rel="stylesheet">
+<script type="text/javascript" src="/assets/webjars/jquery/2.1.3/jquery.js"></script>
+<head>
+
+<body>
+
+<div class="well"> Welcome to Ace's Up. You can win 1/20 times if you play 
+  perfectly. Good Luck! </div>
+
+
+<table align="center">
+    <tr class="title">
+        <td>Column1</td>
+        <td >Column2</td>
+        <td >Column3</td>
+        <td >Column4</td>
+    </tr>
+
+    <tr><td class="columnOfCards" id="c0">
+        <table>
+            <tr><td>
+                <input size="1" maxlength="1" type="text" id="to_col_from_0"  />
+                <button type="button" class="btn btn-primary" id="moveButtoncol0">Move</button>
+        	<br>
+	   </td></tr>
+            <tr><td class="cardLocation l0"></td></tr>
+            <tr><td class="cardLocation l1"></td></tr>
+            <tr><td class="cardLocation l2"></td></tr>
+            <tr><td class="cardLocation l3"></td></tr>
+            <tr><td class="cardLocation l4"></td></tr>
+            <tr><td class="cardLocation l5"></td></tr>
+            <tr><td class="cardLocation l6"></td></tr>
+            <tr><td class="cardLocation l7"></td></tr>
+            <tr><td class="cardLocation l8"></td></tr>
+            <tr><td class="cardLocation l9"></td></tr>
+            <tr><td class="cardLocation l10"></td></tr>
+            <tr><td class="cardLocation l11"></td></tr>
```

Comment:
The Spanish deck only has 40 cards, so you can remove/comment out rows 11-13 for each column.